shaders/outlinepattern.fragment.glsl

```diff
@@ -21,5 +21,13 @@ void main() {
     vec2 pos2 = mix(u_pattern_tl_b, u_pattern_br_b, imagecoord_b);
     vec4 color2 = texture2D(u_image, pos2);
 
-    gl_FragColor = mix(color1, color2, u_mix) * u_opacity;
+    // find distance to outline for alpha interpolation
+    float dist_a = length(v_pos_a - gl_FragCoord.xy);
+    float alpha_a = smoothstep(1.0, 0.0, dist_a);
+
+    float dist_b = length(v_pos_b - gl_FragCoord.xy);
+    float alpha_b = smoothstep(1.0, 0.0, dist_b);
```

Comment:
The `alpha` value here should be based on how far the pixel is from the center of the line. This distance is the same for both patterns so I think we only need one `alpha`.